app/controllers/application_controller.rb

```diff
@@ -16,8 +16,8 @@ def mailer_options
 
   def current_user
     @current_user ||= (
-      User.find_by_auth_token(cookies[:auth_token].to_s) ||
-      User.find_by_user_id(session[:user_id].to_s)
+      User.find_by(auth_token: cookies[:auth_token].to_s) ||
+      User.find_by(id: session[:user_id].to_s)
```

Comment:
Do we still wish to keep `user_id` nomenclature or replace it all?

Reply:
I think the convention _specifically in the session_ makes sense to keep. Just `id` would be nonsensical in a session, though we could do just `user` if you preferred.

I also kept the `user_id` foreign keys because those are actually idiomatic.